test-tools/src/main/java/org/terracotta/utilities/test/logging/ConsoleAppenderCapture.java

```diff
@@ -181,29 +226,88 @@ public Map<String, List<ILoggingEvent>> getLogs() {
         .collect(toMap(Map.Entry::getKey, e -> e.getValue().list));
   }
 
+  /**
+   * Gets the "raw" events logged to the specified target.
+   * <p>
+   * This method is internally synchronized against the {@code Appender} used to capture the logging events.
+   *
+   * @param target the target for which log messages are to be returned
+   * @return the list of events logged to {@code target}l may be empty
+   */
+  @SuppressWarnings("SynchronizationOnLocalVariableOrMethodParameter")
+  public List<ILoggingEvent> getEvents(Target target) {
+    ListAppender<ILoggingEvent> appender = appenderMap.get(target.targetName());
+    if (appender == null) {
+      return emptyList();
+    } else {
```

Comment:
The 'else' can be removed since the 'if' returns.  There's a  few other instances of this same pattern below.

Reply:
You can argue with @chrisdennis about this one.  He dislikes conditionals having alternative actions that aren't expressed as if/then/else ...

Reply:
No need. :)